Studies/python/BayesianOptimizationCustom.py

```diff
@@ -107,6 +117,7 @@ def LoadPoints(self, probed_points_target_output_prev, probed_points_opt_output_
             # if has not been tested before
             if result is not None:
                 raise Exception('target point not allowed: \n{}'.format(json.dumps(p['params'], indent=4)))
+            p = self.UpdateDependencies(p)
```

Comment:
- call it in the beginning of the loop (before `self.FindResults`)
- should be `self.UpdateDependencies(p['params'])`